src/main/java/com/github/pedrovgs/androidwifiadb/view/AndroidDevicesTableModel.java

```diff
@@ -0,0 +1,126 @@
+package com.github.pedrovgs.androidwifiadb.view;
+
+import com.github.pedrovgs.androidwifiadb.model.Device;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.swing.table.AbstractTableModel;
+
+public class AndroidDevicesTableModel extends AbstractTableModel {
+    private static final int COLUMN_DEVICE = 0;
+    private static final int COLUMN_STATE = 1;
+    private static final int COLUMN_ACTION = 2;
+
+    private List<Device> devices;
+
+    public AndroidDevicesTableModel() {
+        devices = new ArrayList<Device>();
+    }
+
+    @Override
+    public String getColumnName(int column) {
+        String value;
+        switch (column) {
+            case COLUMN_DEVICE:
+                value = "Device";
+                break;
+            case COLUMN_STATE:
+                value = "State";
+                break;
+            case COLUMN_ACTION:
+                value = "Action";
+                break;
+            default:
+                return null;
```

Comment:
Instead of return null can't we log a trace and return the original value defined few lines above?